src/main/java/zeenea/connector/property/PropertyDefinition.java

```diff
@@ -108,6 +121,7 @@ public static LongTextPropertyDefinition longText(String code) {
    * @param label the label of the property definition
    * @return a new LongTextPropertyDefinition instance
    */
+  @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Idem : 
```suggestion
  Et là également : 
```suggestion
  
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use longText(name) instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```
```